src/plopp/backends/matplotlib/canvas.py

```diff
@@ -46,6 +46,10 @@ class Canvas:
         The maximum value for the vertical axis. If a number (without a unit) is
         supplied, it is assumed that the unit is the same as the current vertical axis
         unit.
+    autoscale:
+        The behavior of the axis limits. If ``auto``, the limits automatically
+        adjusts every time the data changes. If ``grow``, the limits are allowed to
+        grow with time but they do not shrink.
```

Comment:
Would a third option make sense: Computing vmin/vmax and setting a fixed scale. I know we are not doing this automatically due to the cost (even though I am not sure about that argument any more --- it was indeed a problem with event data and dynamic resampling, but is it still an actual problem?), but having it as a "costly" non-default option may be useful?

Edit: I now realize that this may not be possible in the Plopp architecture, as the plot does never see the higher-dimensional data in case of `slicer`. So maybe `slicer` should have that additional `autoscale` option (and set `vmin/vmax` based on that?

Reply:
Yes, it's the reason you list in your edit. We could add it to `slicer`, i'd be happy with that.